queries/unison/highlights.scm

```diff
@@ -12,33 +12,30 @@
 (literal_boolean) @boolean
 
 ; Keywords
+(type_kw) @keyword.type
+
 [
   (kw_forall)
-  (unique)
-  (type_kw)
   (do)
   (kw_let)
   (ability)
   (where)
 ] @keyword
 
-(kw_equals) @keyword.operator
+(kw_equals) @operator
 
 (structural) @keyword.modifier
 
 (unique) @keyword.modifier
 
 (type_constructor) @constructor
 
-(doc_block) @comment.documentation
```

Comment:
This should stay as well, in case people do not want injections.

Reply:
If needed, set a lower priority here.

Reply:
How do I set a lower priority?

Reply:
With #set! priority directive (on mobile right now)

Reply:
Thank you! I found some examples and took it from there.